packages/app/src/views/AddModule/wizards/OzGovernor/service/moduleDeployment.ts

```diff
@@ -3,6 +3,8 @@ import { ethers } from "ethers"
 import { enableModule, TxWitMeta } from "services"
 import SafeAppsSDK from "@gnosis.pm/safe-apps-sdk"
 
+const SECONDS_IN_A_BLOCK = 12
```

Comment:
I don't think we can hardcode this, since ~12 seconds per block is only true for mainnet. Other networks have different block times. Rather, we should let the user choose the number of blocks directly, then we could (optionally) give an estimate based on the block times for this network.

Reply:
Thank you!